frontend/src/composables/credential/helper.js

```diff
@@ -4,14 +4,46 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+export function isSecretBinding (binding) {
+  return binding?.kind === 'SecretBinding'
+}
+export function isCredentialsBinding (binding) {
+  return binding?.kind === 'CredentialsBinding'
+}
+
 export function isSharedCredential (binding) {
-  const bindingNamespace = binding.metadata.namespace
+  const bindingNamespace = binding?.metadata.namespace
   let refNamespace
-  if (binding._isSecretBinding) {
-    refNamespace = binding.secretRef.namespace
-  } else if (binding._isCredentialsBinding) {
-    refNamespace = binding.credentialsRef.namespace
+  if (isSecretBinding(binding)) {
+    refNamespace = binding?.secretRef.namespace
+  } else if (isCredentialsBinding(binding)) {
+    refNamespace = binding?.credentialsRef.namespace
```

Comment:
there is a `credentialsName` computed property / function, why not also have a credentialsNamespace computed property and/or function